pkg/cmd/workspace/util/repository_wizard.go

```diff
@@ -114,19 +160,50 @@ func getRepositoryFromWizard(config RepositoryWizardConfig) (*apiclient.GitRepos
 	}
 
 	var providerRepos []apiclient.GitRepository
-	err = views_util.WithSpinner("Loading", func() error {
-		providerRepos, _, err = config.ApiClient.GitProviderAPI.GetRepositories(ctx, providerId, namespaceId).Execute()
-		return err
-	})
-
-	if err != nil {
-		return nil, err
-	}
+	var chosenRepo *apiclient.GitRepository
+	page = 1
+	perPage = 100
 
 	parentIdentifier := fmt.Sprintf("%s/%s", providerId, namespace)
-	chosenRepo := selection.GetRepositoryFromPrompt(providerRepos, config.ProjectOrder, config.SelectedRepos, parentIdentifier)
-	if chosenRepo == nil {
-		return nil, common.ErrCtrlCAbort
+	for {
+		// Fetch repos for the current page
+		err = views_util.WithSpinner("Loading Repositories", func() error {
+
+			repos, _, err := config.ApiClient.GitProviderAPI.GetRepositories(ctx, providerId, namespaceId).Page(page).PerPage(perPage).Execute()
+			if err != nil {
+				return err
+			}
+			curPageItemsNum = len(repos)
+			providerRepos = append(providerRepos, repos...)
```

Comment:
After loading more results, the list should focus on the first new element in the list.

Currently, after loading more, it's always reset to the start which is problematic if you have to scroll 100+ repos every time.

Reply:
done